.pre-commit-config.yaml

```diff
@@ -7,6 +7,9 @@ repos:
     hooks:
       - id: trailing-whitespace
         exclude: tests(/\w*)*/functional/t/trailing_whitespaces.py|tests/pyreverse/data/.*.html|doc/data/messages/t/trailing-whitespace/bad.py
+      #      - id: file-contents-sorter # commented out because it does not preserve comments order
```

Comment:
Should we just not add it then?

Reply:
It's still useful to sort it and then ``git add -p`` the changes.

Reply:
Yeah but I don't think we should be committing code that is commented out to `main` right?

Reply:
It prevents us from having to recreate the sorting from zero every time we want to sort the file, Is there any downside apart from the file being a little longer ?

Reply:
Hmm I think it looks a bit sloppy to have code that you don't intend to use, but I guess it is fine?